util/util.go

```diff
@@ -377,6 +379,23 @@ func logIfNotErrno(err error, what string, ignores ...syscall.Errno) (logged boo
 	return true
 }
 
+// LogJson is a helper function to log data structures as JSON.
+// The given wrappingKey will contain the provided data.
+// Unexpected characters will be replaced by cats.
+// The logged lines then may be post-processed with external pretty log parsers.
+func LogJson(wrappingKey string, data interface{}) {
+	rawResult, err := json.Marshal(data)
+	if err != nil {
+		logrus.Fatal(err)
+	}
+	// Replace bad chars with a cat for external pretty log parsers
+	re := regexp.MustCompile(`[#=\']|\\"`)
+	safeResult := string(re.ReplaceAll([]byte(rawResult), []byte("🐱")))
```

Comment:
I love [cats](https://twitter.com/TSweeneyRedHat/status/1309933044942475265), but is a cat symbol really generally used?  I was expecting an asterisk ("*") or a similar less fun character.

Reply:
done